### PR TITLE
:hammer: Storybook gh pages

### PR DESCRIPTION
**Pull request by cbuv**
## What does this pull request change?
publish apollo-components storybook to github pages
Storybook available at https://equinor.github.io/apollo-react-components/

## Why is this pull request needed?
quickly explore this library

### CODE REVIEW

File: .github/workflows/storybook.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+name: Build and Deploy Storybook
+
+on: [push]
+#on:
+#  push:
+#    branches:
+#      - main
```
Comment (mholta) [inline .github/workflows/storybook.yml:7]: Comment out

File: .github/workflows/storybook.yml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+jobs:
+  build-and-deploy:
+    concurrency: ci-${{ github.ref }} # Recommended if you intend to make multiple deployments in quick succession.
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout 🛎️
+        uses: actions/checkout@v3
+
+      - name: Install and Build ⬇️ # This example project is built using npm and outputs the result to the 'build' folder. Replace with the commands required to build your project, or remove this step entirely if your site is pre-built.
```
Comment (mholta) [inline .github/workflows/storybook.yml:24]: Remove comment?

**Review by mholta — CHANGES_REQUESTED** [review]
Looks good!

**Comment by mholta** [comment]
Also, please add changeset

File: .github/workflows/storybook.yml (hunk 2 above)
Comment (cbuv) [reply .github/workflows/storybook.yml]: agree, removed in [380cf19](https://github.com/equinor/apollo-react-components/pull/41/commits/380cf199659a8d40b26b02ed2a683153cf70f21e)

File: .github/workflows/storybook.yml (hunk 1 above)
Comment (cbuv) [reply .github/workflows/storybook.yml]: reverted in [380cf19](https://github.com/equinor/apollo-react-components/pull/41/commits/380cf199659a8d40b26b02ed2a683153cf70f21e)

**Comment by cbuv** [comment]
> Also, please add changeset

I don't see any reasons to add a changeset for this change, since there are no changes to any package.

**Review by mholta — APPROVED** [review]
LGTM

**Pull request merged**